frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -20,6 +32,20 @@ export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2T
             width={'max(40vw)'}
         >
             <div>
+                {includesPipelinesAddon && itemsToDisable.length > 0 ? (
+                    <div className="mb-6">
+                        <div className="mb-4">
+                            <h3 className="mt-2 mb-2">{`Important: Disable remaining export apps`}</h3>
+                            <p>
+                                To avoid unexpected impact on your data, you must explicitly disable the following apps
+                                and exports before unsubscribing:
+                            </p>
+                        </div>
+                        <ExportsUnsubscribeTable />
+                    </div>
+                ) : (
+                    <></>
+                )}
                 <h3 className="mt-2 mb-4">{`Why are you unsubscribing from ${product.name}?`}</h3>
```

Comment:
Need to constrain the width of this box because it overlaps with the close X on smaller screens. Ref the snapshots.